udata/core/reuse/models.py

```diff
@@ -33,15 +35,27 @@ def check_url_does_not_exists(url):
         raise FieldValidationError(_("This URL is already registered"), field="url")
 
 
+class ReuseBadge(Badge):
+    kind = db.StringField(required=True, choices=list(BADGES.keys()))
+
+
+class ReuseBadgeMixin(BadgeMixin):
+    badges = field(BadgesList(ReuseBadge), **BadgeMixin.default_badges_list_params)
+    __badges__ = BADGES
+
+
 @generate_fields(
     searchable=True,
-    additionalSorts=[
+    additional_sorts=[
         {"key": "datasets", "value": "metrics.datasets"},
         {"key": "followers", "value": "metrics.followers"},
         {"key": "views", "value": "metrics.views"},
     ],
+    additional_filters=[
+        "organization.badges",
```

Comment:
I think this filter is called with `organization_badges` instead of `organization_badge` with the *api fields system* :tm:

Reply:
I'm afraid I don't understand what you mean? The "api fields system" splits on the dot, not on an underscore, and `Organization` has a `badges` field, with the final `s`.

Am I missing something?

Reply:
I think we have `api/1/datasets/?organization_badge=<foo>` VS `api/1/reuses/?organization_badges=<foo>`  for example

Reply:
Woah you're right! Good catch!